src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -1310,6 +1310,8 @@ protected virtual Task PerformHealthCheck(ServerNode serverNode, int nodeIndex,
         protected Task _firstTopologyUpdate;
         protected string[] _lastKnownUrls;
         private readonly DisposeOnce<ExceptionRetry> _disposeOnceRunner;
+        private TimeSpan? _sendTimeout;
```

Comment:
can be removed?

Reply:
probably yes. I will move Request/Response timout to Conventions I think